spikeinterface_gui/tests/test_mainwindow.py

```diff
@@ -82,15 +87,12 @@ def test_mainwindow(start_app=False, verbose=True, curation=False, only_some_ext
     else:
         win = run_mainwindow(
             analyzer,
-            # backend="qt",
```

Comment:
if wwe split in 2 testing files then we need renaming and better code sharing instead of a copy/paste no ?